build.gradle

```diff
@@ -59,7 +59,20 @@ subprojects {
 
     spotless {
         java {
+            removeUnusedImports()
```

Comment:
It looks like the `googleJavaFormat().aosp()` already removes unused imports.  I don't think we need this line.